File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.ts
function removeDuplicates(nums: number[]): number {
    let x: number = 0

    for (let i: number = 0; i < nums.length; i++) {
        if (nums[i] !== nums[i + 1]) {
            nums[x++] = nums[i]
        }
    }
    return x
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        uniqueCount = 1

        for i in range(1, len(nums)):
            if nums[i] != nums[i-1]:
                nums[uniqueCount] = nums[i]
                uniqueCount += 1

        return uniqueCount